src/components/common/NavHeader.js

```diff
@@ -33,6 +35,15 @@ const NavHeader = () => {
           <Link to="/services" />
           Services
         </Menu.Item>
+        <Menu.Item
+          key="logout"
+          onClick={() => {
+            authService.logout();
```

Comment:
How does authService.logout() work? Is there a token or cookie that is cleared from the browser when logging out?

Reply:
This revokes the current okta session from my understanding of the okta documentation.